apps/client/components/Form/Input.tsx

```diff
@@ -0,0 +1,40 @@
+import clsx from 'clsx'
+
+interface Props {
+  readonly children?: React.ReactNode
+  readonly className?: string
+  readonly error?: React.ReactNode
+  readonly icon: JSX.Element
+  readonly label: string
+  readonly type: string
+  readonly validate?: any
+}
+
+export const Input = ({ children, className, error, icon, label, type, validate }: Props) => {
+  return (
+    <div className={clsx('flex flex-col gap-[0.7rem] w-full', className)}>
+      {children ? (
+        children
```

Comment:
To imo słabe, jeżeli ten komponent wygląda i zachowuje się kompletnie inaczej w zależności od tego czy są children czy nie to, to powinny być to dwa osobne komponenty. Jeżeli reużywalny jest sam wrapper to on może być osobnym komponentem.